c_models/src/my_models/implementations/my_full_neuron_impl.h

```diff
@@ -93,7 +93,8 @@ static void neuron_impl_do_timestep_update(
 }
 
 #if LOG_LEVEL >= LOG_DEBUG
-void neuron_impl_print_inputs(uint32_t n_neurons) {
+__attribute__((unused)) // Marked unused as only used sometimes
```

Comment:
Are we able to use the #define for this that sPyNNaker uses?

Reply:
Looks like it, yes - done.